docs/openapi.yaml

```diff
@@ -1,791 +1,1564 @@
-                type: object
-                properties:
-                  userId:
-                    description: Unique ID of the Plan
-                    type: integer
-                    format: int64
-                    example: 72
-        '400':
-          description: Invalid request body
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
+        '200':
+          $ref: '#/components/responses/Plan_DELETE_200'
+        '401':
+          $ref: '#/components/responses/401'
         '403':
-          description: Request not made by an authorized user
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
         '500':
-          description: Internal server error
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
+          $ref: '#/components/responses/500'
 
-  '/plan/{planId}':
+
+  '/plan/{planId}/similar':
     get:
-      summary: Fetch data about a specific applied Plan
+      summary: Fetch the number of applied Plans similar to the Plan
       description: >
-        Returns information about a specific applied Plan with the provided
-        `planId`.
-      operationId: getPlanById
+        Returns a count of accepted similar Plans and a count of rejected
+        similar Plans.
+      operationId: getSimilarPlans
       tags:
         - Plan
       parameters:
-        - name: planId
-          in: path
-          description: ID of the plan to fetch
-          required: true
-          schema:
-            type: integer
-            format: int64
-          example: 72
+        - $ref: '#/components/parameters/Plan_GET_PlanId'
       responses:
         '200':
-          description: Success
+          $ref: '#/components/responses/Plan_Similar_GET_200'
+        '401':
+          $ref: '#/components/responses/401'
         '403':
-          description: Request not made by an authorized user
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
+          $ref: '#/components/responses/403'
         '404':
-          description: Plan with the specified `planId` not found
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
+          $ref: '#/components/responses/404'
         '500':
-          description: Internal server error
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
+          $ref: '#/components/responses/500'
 
-    patch:
-      summary: Partially update a specific applied Plan
+  '/plan/search/{text}/{status}/{sort}/{order}/{cursorPrimary}/{cursorSecondary}':
```

Comment:
Example parameters seem to have some sort of formatting issue.

![Capture1](https://user-images.githubusercontent.com/32185302/81131423-f7000600-8eff-11ea-8cc1-172f18998334.JPG)